dev/buildbundle-veloxbe.sh

```diff
@@ -4,7 +4,19 @@ BASEDIR=$(dirname $0)
 source "$BASEDIR/builddeps-veloxbe.sh"
 
 cd $GLUTEN_DIR
+
+DISTDIR="$GLUTEN_DIR/dist"
+rm -rf "$DISTDIR"
+mkdir -p "$DISTDIR"
+
 mvn clean package -Pbackends-velox -Prss -Pspark-3.2 -DskipTests
+cp package/target/gluten-*-bundle-spark*.jar "$DISTDIR"
```

Comment:
should we use `mv` ? It will make binary size bigger

Reply:
> should we use `mv` ? It will make binary size bigger

The `package/target` directory is just the mvn build output directory of `package` module, I think we don't need to care about its size, it will be cleaned after `mvn clean`.